docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       - dbus-session
       - diagnostics
     environment:
-      - FIRMWARE_VERSION=2021.11.17.2-1
+      - FIRMWARE_VERSION=2021.11.17.2-2
```

Comment:
This needs to get rebumped because of: https://github.com/NebraLtd/helium-miner-software/pull/242

I'll do this when I have a sec.

Reply:
Updated but forgot to resolve merge conflicts 🤦

Reply:
Closed in favour of #243